DESCRIPTION

```diff
@@ -33,4 +33,4 @@ Suggests:
     rmarkdown,
     R.rsp,
     sp
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.0
```

Comment:
What is best practice here? If I ran `roxygenise()` on a more recent version than in the `DESCRIPTION` the `DESCRIPTION` should be updated right?

Or should I leave the `roxygenise()` step up to you (maintainer) so they can handle the version dependency themselves?